app/assets/javascripts/controllers/middleware_server/middleware_server_controller.js

```diff
@@ -25,17 +25,18 @@ MwServerGroupController.$inject = ['$scope', 'miqService' ];
  * parent/child controller relationships.
  * @param $scope
  * @param miqService
+ * @param mwAddDatasourceService
  * @constructor
  */
-function MwServerController($scope, miqService) {
-  return MwServerControllerFactory($scope, miqService, false);
+function MwServerController($scope, miqService, mwAddDatasourceService) {
+  return MwServerControllerFactory($scope, miqService, mwAddDatasourceService, false);
```

Comment:
That comes from `function MwServerControllerFactory($scope, miqService, mwAddDatasourceService, isGroupDeployment)` so it is just a flag to tell the factory whether we are dealing with Servers or ServerGroups while maintaining the same Factory signature.